src/batch/src/executor/join/distributed_lookup_join.rs

```diff
@@ -394,7 +395,13 @@ impl<S: StateStore> LookupExecutorBuilder for InnerSideExecutorBuilder<S> {
         } else {
             let iter = self
                 .table
-                .batch_iter_with_pk_bounds(self.epoch.clone().into(), &pk_prefix, .., false)
+                .batch_iter_with_pk_bounds(
+                    self.epoch.clone().into(),
+                    &pk_prefix,
+                    ..,
+                    false,
+                    PrefetchOptions { exhaust_iter: true },
```

Comment:
Let's user constructor to construct `PrefetchOptions`. Otherwise, we still need to change all executor codes in the future when adding more fields. We can have two constructors for now:
- `PrefechOptions::new_for_exhaust_iter`
- `PrefetchOptions::default`

Reply:
> Let's user constructor to construct `PrefetchOptions`. Otherwise, we still need to change all executor codes in the future when adding more fields. We can have two constructors for now:
> 
> * `PrefechOptions::new_for_exhaust_iter`
> * `PrefetchOptions::default`

fixed